packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -315,6 +312,24 @@ async function createLanguageClient(extensionContext: vscode.ExtensionContext) {
 
     languageClientUtils.setClientInstance(languageClient);
     await languageClient!.start();
+    // Client is running
+
+    // The listener should be set after languageClient is ready
+    // Language client will get notified once async init jobs are done
+    languageClient.onNotification('indexer/done', async () => {
+      await getTestOutlineProvider().refresh();
+      languageServerReady();
+    });
+
+    // client will check status with server after initialization
+    const indexerStatus = await languageClient.sendRequest('indexer/status');
```

Comment:
I would like to "hide" the details of new endpoints. What do you think about adding a new function in ApexLanguageClient that provides access to the endpoint, i.e. ApexLanguageClient#getServerState() and possible a convienence function of ApexLanguageClient#isRunning(). This will make the call available to others that might be concerned with the language server state.